Makefile

```diff
@@ -68,6 +68,12 @@ endif
 # tools. (i.e. podman)
 CONTAINER_TOOL ?= podman
 
+
+# Use the image urls from the yaml that is used with Kustomize for local
+# development.
+DPU_OPERATOR_IMAGE := $(shell yq -r .spec.template.spec.containers[0].image config/dev/local-images.yaml)
```

Comment:
Conventions is to wrap yq to $(yq) and add a make rule to install yq. However I am ok with this.